packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/SingleImageViewer/SingleImageViewer.js

```diff
@@ -71,13 +71,14 @@ function SingleImageViewer({
         overflow='hidden'
         width='100%'
       >
-        {title?.id && title?.text && (
-          <title id={title.id}>{title.text}</title>
-        )}
```

Comment:
`<title>` in HTML can only be used inside `<head>`, and generates the title of the page for browser tabs, browser history, search engines etc.